frontend/packages/pipelines-plugin/console-extensions.json

```
@@ -10,6 +10,326 @@
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1alpha1",
+        "kind": "Condition"
+      },
+      "component": { "$codeRef": "conditionsComponent.ConditionListPage" }
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1beta1",
+        "kind": "Task"
+      },
+      "component": { "$codeRef": "tasksComponent.TaskListPage" }
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1beta1",
+        "kind": "ClusterTask"
+      },
+      "component": { "$codeRef": "clusterTasksComponent.ClusterTaskListPage" }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/pipelines", "/tasks", "/triggers", "/dev-pipelines"],
+      "component": {
+        "$codeRef": "pipelineComponent.NamespaceRedirect"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": false,
+      "path": ["/pipelines/all-namespaces", "/pipelines/ns/:ns"],
+      "component": {
+        "$codeRef": "pipelinesListsComponent.PipelinesListsPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": false,
+      "path": ["/tasks/all-namespaces", "/tasks/ns/:ns"],
+      "component": {
+        "$codeRef": "tasksComponent.TasksListsPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": false,
+      "path": ["/triggers/all-namespaces", "/triggers/ns/:ns"],
+      "component": {
+        "$codeRef": "triggersListsComponent.TriggersPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "perspective": "dev",
+      "path": [
+        "/k8s/all-namespaces/tekton.dev~v1beta1~Pipeline",
+        "/k8s/ns/:ns/tekton.dev~v1beta1~Pipeline"
+      ],
+      "component": {
+        "$codeRef": "pipelinesComponent.PipelinesPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": "/k8s/ns/:ns/tekton.dev~v1beta1~Pipeline/~new/builder",
+      "component": {
+        "$codeRef": "pipelinesComponent.PipelineBuilderPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": "/k8s/ns/:ns/tekton.dev~v1beta1~Pipeline/:pipelineName/builder",
+      "component": {
+        "$codeRef": "pipelinesComponent.PipelineBuilderEditPage"
+      }
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "perspective": "dev",
+      "path": [
+        "/k8s/all-namespaces/tekton.dev~v1alpha1~PipelineRun",
+        "/k8s/ns/:ns/tekton.dev~v1alpha1~PipelineRun"
+      ],
+      "component": {
+        "$codeRef": "pipelineRunsComponent.PipelineRunsPage"
```

based on my previous comment. (update in all other occurrences in this file)

```suggestion
        "$codeRef": "pipelineComponents.PipelineRunsPage"
```